data/templates/install-photon/photon-os-ks

```diff
@@ -49,6 +49,11 @@
     "postinstall": [
                     "#!/bin/sh",
 
+                    "# notify the current progress",
+                    <%_ if( typeof progressMilestones !== 'undefined' && progressMilestones.postConfigUri ) { _%>
```

Comment:
I intentionally did this. Though it doesn't align with other file, but it aligns within the file itself. I think keep the whole file in uniform style is important, and I don't want to refactor all files into same style.